src/api/server.ts

```diff
@@ -2,7 +2,7 @@ import app from './express';
 import dotenv from 'dotenv';
 dotenv.config();
 
-const port = process.env.PORT || 3006;
+const port = process.env.API_PORT || 3006;

```

Comment:
docker-compose has this listed as defaulted to 3000 I think? do we want to make these consistent?

Reply:
Feedback taken and addressed in latest commit